modified readme and SEO improvements

ananyo2012 opened the pull request:
Created first pull request.Modified readme.

sonalkr132 commented:
Thanks @ananyo2012 

Can you please update this so that readme has information about all the example file? Besides devise, I think there is cloudinary.rb and production.rb

ananyo2012 commented:
I have added new commits to my previous pull request.changed  the title and meta data and also modified the readme as you said.

sonalkr132 commented on app/views/users/passwords/edit.html.erb:
```diff
@@ -1,4 +1,5 @@
-<h2>Change your password</h2>
+<% @pageTitle = "Set new Password | CAT Forum" %>
+<% @PageDescription = "Set new password for your account" %><h2>Change your password</h2>
 
```
why did you remove content of h2 tag here?

ananyo2012 replied:
please see beside  `<% @PageDescription = "Set new password for your account" %>` in line 2.the h2 tag is after the erb tag.

sonalkr132 replied:
Please fix the indentation. Now that many people are working on same
project it is important that we try to keep our code clean.
On 27-May-2015 11:04 pm, "Ananya Maiti" notifications@github.com wrote:

> In app/views/users/passwords/edit.html.erb
> https://github.com/sonalkr132/cat-forum/pull/2#discussion_r31159134:
> 
> > @@ -1,4 +1,5 @@
> > -<h2>Change your password</h2>
> > +<% @pageTitle = "Set new Password | CAT Forum" %>
> > +<% @PageDescription = "Set new password for your account" %><h2>Change your password</h2>
> 
> please see beside <% @PageDescription = "Set new password for your
> account" %> in line 2.the
> tag is after the erb tag.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/sonalkr132/cat-forum/pull/2/files#r31159134.

ananyo2012 commented:
no i didn't remove it actually.it is there but not indented properly.the changes are in the same line actually.i didn't observe it before properly so i didn't give the indentation well.

sonalkr132 commented:
Can you please point me to the line where it says 'change your password' under h2 tag? As far as I can see it is not there https://github.com/ananyo2012/cat-forum/blob/github_master/app/views/users/passwords/edit.html.erb

ananyo2012 commented:
added microdata and modified the robots.txt to disallow the admin area of cat-forum.also fixed the previous indentation and changed the name of the image cat.jpg to CAT-Forum .jpg and made the necessary changes.

sonalkr132 commented:
@ghoshbishakh @ananyo2012 Just so that everyone knows. We will be using 2 spaces (not tab) for indentation. Configure your editor accordingly.

ananyo2012 commented:
ok will be taking care for next pull requests
